index.js

```diff
@@ -149,6 +150,90 @@ class GithubScm extends Scm {
     }
 
     /**
+     * Adds the Screwdriver webhook to the Github repository
+     * @method _addWebhook
+     * @param  {Object}    config            Config object
+     * @param  {String}    config.scmUri     The SCM URI to add the webhook to
+     * @param  {String}    config.token      Service token to authenticate with Github
+     * @param  {String}    config.webhookUrl The URL to use for the webhook notifications
+     * @return {Promise}                     Resolve means operation completed without failure.
+     */
+    _addWebhook(config) {
+        const searchForHook = (scmInfo, url, page) => {
+            const getHooks = this.breaker.runCommand({
+                action: 'getHooks',
```

Comment:
Even if the repository is public, you need elevated privileges to view its webhooks. 

For my test token, these are the settings I enabled so that it can modify the hooks: 

![screen shot 2016-12-02 at 9 16 33 pm](https://cloud.githubusercontent.com/assets/721148/20857019/afe88efe-b8d4-11e6-8174-4f73f1917b6a.png)